docker/Dockerfile

```
@@ -14,7 +14,7 @@ ENV PYTHONPATH=/app
 # Install system dependencies required for OpenCV
 RUN apt-get update && apt-get install -y --no-install-recommends \
     libgl1-mesa-glx=22.3.6-1+deb12u1 \
-    libglib2.0-0=2.74.6-2+deb12u6 \
+    libglib2.0-0=2.74.6-2+deb12u7 \
```

```suggestion
    libglib2.0-0=2.74.* \
```
As in Geti -> https://github.com/open-edge-platform/geti/blob/683b022770edca8dd243ec3fa0cc04bc74a7d4b6/interactive_ai/workflows/train/Dockerfile#L43